### PR TITLE
Borging Antagonists

### PR DESCRIPTION
**Pull request by Ikarrus**
- Cyborgs no longer get a message appended to their memory telling them their objectives are null and void, which was no longer the case anyways.
- Borged and AIzed antagonists will get this message: **You must obey your silicon laws and master AI above all else. Your objectives will consider you to be dead.**
- Removed restriction on borging head revs and gang bosses (They get deconverted instead)
- Brains are set to always appear on top of human icons, so they can be more easily found, especially during surgery.

### CODE REVIEW

**Comment by Incoming5643** [comment]
but they do

**Comment by Ikarrus** [comment]
You should see the line directly above the ones I added

"As a cyborg, any objectives listed here are null and void, and will be marked as failed. They are simply here for memory purposes."

**Comment by Incoming5643** [comment]
I'm just saying that mechanically that's not true.

**Comment by veganzombeh** [comment]
I agree with @Incoming5643 here. The message is slightly misleading; not all objectives would necessarily be failed.

**Comment by Lumipharon** [comment]
Yeah, I don't see anythin gained from deantaging silicons.
They have their laws to keep them in check, but if they can try to achieve whatever objectives they have, within the laws that they have, that's neat as fuck.

**Comment by Incoming5643** [comment]
Follow your dreams :rainbow:

**Comment by Ikarrus** [comment]
This is starting to touch on policy so I'll grab the headmin opinion on this.

**Comment by Ikarrus** [comment]
tbh I dont care either way I just want it to be consistent with one or the other.

File: code/game/objects/items/robot/robot_parts.dm [hunk 1]
```diff
@@ -226,6 +226,8 @@
 
 			if(O.mind && O.mind.special_role)
 				O.mind.store_memory("As a cyborg, any objectives listed here are null and void, and will be marked as failed. They are simply here for memory purposes.")
```
Comment (Incoming5643) [inline code/game/objects/items/robot/robot_parts.dm]: Technically removing this line is more proper than adding the others, as borgs antags are still antags in every code sense of the word.

A line saying that laws come before objectives isn't a bad idea though.

**Comment by Ikarrus** [comment]
How binding do we want the AI slavery to be, then?

Would a purged cyborg be permitted by admins to kill the AI to achieve their objective?

And we need to consider how this would discourage security from borging their prisoners even more than they are already discouraged from.

**Comment by Ikarrus** [comment]
@KorPhaeron @hornygranny @stickymayhem @sometinyprick Your opinions, please.

**Comment by Incoming5643** [comment]
To be clear here, the current code will clear "group antag status(es)" such as revolutionary or cult status, but this happened when the brain is put in the MMI. 

Singular antag statuses like a traitor or a wizard remain throughout the borging processes and are still "active" at roundend in terms of reporting. They are reported as dead, but can still complete objectives unrelated to being alive. 

"Die a glorious death" antags can still greentext as borgs if they did not have a steal objective (or that objective was a jetpack and they're a mining borg with the jetpack attachment).

**Comment by Ikarrus** [comment]
Is it just the wording that bothers you then?

**Comment by Incoming5643** [comment]
it's that the wording doesn't reflect the reality of the situation, but I guess it does reflect the reality of the situation in relation to how the admins handle it.

I was mostly just pointing that out here for any headmins who read it who might not know the exact mechanic in play here.

**Comment by Lumipharon** [comment]
I play sec a lot, and I can't see why that would make me want to borg antags less. I'm not making them unslaved, purged borgs, they've got laws that most likely prevent them from murdering me (asimov, paladin and robo cop would all stop that). And borgs being anags is irrelevent if they get subverted anyway.

**Comment by PKPenguin321** [comment]
>Would a purged cyborg be permitted by admins to kill their Master AI to achieve their objective?

a purged silicon is basically just a traitor in a robot body with no limitations, a traitor cyborg in other words
and in that case they could kill their master AI, flood plasma, deconstruct the cyborg control console, and kill everything if they were so inclined

**Comment by Ikarrus** [comment]
Updated. Check the PR description.

**Comment by Ikarrus** [comment]
<Ikarrus> KorPhaeron would a purged cyborg be allowed to kill their master AI?
<KorPhaeron> I don't think so
<KorPhaeron> borgs follow their AIs outside of their laws
<KorPhaeron> there is no law in the asimov stuff that says "follow the AI"
<KorPhaeron> but they still do

**Comment by Fox-McCloud** [comment]
and here I was hoping this PR was going to add the opt-in autoborger talked about months ago.

**Comment by Lumipharon** [comment]
neat.

**Pull request merged**